driver/src/main/scala/core/nodeset.scala

```diff
@@ -667,6 +655,53 @@ final class ChannelFactory private[reactivemongo] (
     pipeline
   }
 
+  private def sslContext = {
+    import java.io.FileInputStream
+    import java.security.KeyStore
+    import javax.net.ssl.{ KeyManagerFactory, TrustManager }
+
+    val keyManagers = Option(System.getProperty("javax.net.ssl.keyStore")).map { path =>
+
+      val password = Option(System.getProperty("javax.net.ssl.keyStorePassword")).getOrElse("")
```

Comment:
That's all done now, mutual-ssl and self-ssl jobs are passing, a couple have timed out but I'm restarting them